hack/postrelease/test_helm_artifacts.py

```diff
@@ -20,7 +20,7 @@ def test_calico_release_in_helm_index():
         % (RELEASE_VERSION, RELEASE_VERSION)
     )
 
-    req = requests.get("https://projectcalico.docs.tigera.io/charts/index.yaml")
```

Comment:
We should still expect the old URL to work - we have redirects set up and should have a test to make sure that they continue to work.

Reply:
i'll revert